docs/root/configuration/upstream/cluster_manager/cluster_stats.rst

```diff
@@ -90,8 +90,11 @@ Every cluster has a statistics tree rooted at *cluster.<name>.* with the followi
   upstream_rq_rx_reset, Counter, Total requests that were reset remotely
   upstream_rq_tx_reset, Counter, Total requests that were reset locally
   upstream_rq_retry, Counter, Total request retries
+  upstream_rq_retry_active, Gauge, Total number of requests with active retries
   upstream_rq_retry_backoff_exponential, Counter, Total retries using the exponential backoff strategy
+  upstream_rq_retry_backoff_exponential_active, Gauge, Total retries currently in backoff using the exponential backoff strategy
   upstream_rq_retry_backoff_ratelimited, Counter, Total retries using the ratelimited backoff strategy
+  upstream_rq_retry_backoff_ratelimited_active, Gauge, Total retries currently in backoff using the ratelimited backoff strategy
```

Comment:
These, in combination with what's already there, should be enough for users to calculate their budget totals and usage against the budget on their own. This is helpful since the remaining gauge isn't implemented for retry budgets.